css/style.css

```diff
@@ -1,6 +1,881 @@
+    margin: 42px 0 0 224px;
+  }
+
+  .sneaker-item__name {
+    margin: 0 0 9px 0;
+  }
+
+  .shop-wrap {
+    margin-bottom: 75px;
+  }
+
+  .colour-circle {
+    width: 45px;
+    padding: 0 16px 0 0;
+  }
+
+  /* 2 секция */
+
+  .history {
+    height: 442px;
+    margin-bottom: 92px;
+  }
+
+  .history__video {
+    max-width: 578px;
+    padding: 4px 0 0 13px;
+  }
+
+  .history__wrap {
+    max-width: 246px;
+    height: 182px;
+    margin: 0 19px 0 0;
+  }
+
+  .history__title {
+    margin: 0 0 0 0;
+  }
+
+  .history__text {
+    font-size: 13px;
+    line-height: 16px;
+    margin: 0 0 0 0;
+  }
+
+  .arrow-white_img2 {
+    padding-left: 11px;
+  }
+}
+
+@media (min-width: 769px) and (max-width: 992px) {
+  .conteiner {
+    max-width: 890px;
+  }
+
+  .header-menu {
+    margin: 0 0 0 0;
+  }
+
+  .sneaker-item::after {
+    display: none;
+  }
+}
+
+@media (max-width: 768px) {
+  .header {
+    height: 114px;
+  }
+
+  .header__column {
+    height: auto;
+    margin-top: 5px;
+  }
+
+  .header__logo_tabimg {
+    display: block;
+    margin: 0 0 0 42px;
+  }
+
+  .header__burger {
+    display: block;
+    position: relative;
+    width: 60px;
+    height: 24px;
+    margin: 0 45px 0 0;
+  }
+
+  .header__burger::before,
+  .header__burger::after {
+    content: '';
+    background-color: #000;
+    position: absolute;
+    width: 100%;
+    height: 9px;
+  }
+
+  .header__burger::before {
+    top: 0;
+  }
+
+  .header__burger::after {
+    bottom: 0;
+  }
+
+  .header__logo_img,
+  .header-menu,
+  .search-form,
+  .right-menu {
+    display: none;
+  }
+
+  /* 1 секция */
+
+  .imageblock {
+    background: url(../image/rob-potter-768.svg);
+    height: 416px;
+    margin-bottom: 63px;
+    background-size: cover;
+  }
+
+  .imageblock__title {
+    font-size: 50px;
+    line-height: 48px;
+    width: 320px;
+    margin: 0 0 46px 0;
+    padding: 120px 0 0 45px;
+  }
+
+  .imageblock__link {
+    padding: 0 0 0 45px;
+  }
+
+  .link-arrow__white {
+    font-size: 20px;
+    line-height: 19px;
+    font-weight: bold;
+  }
+
+  .arrow-white__tabimg {
+    display: inline-block;
+    padding: 3px 0 0 5px;
+  }
+
+  .arrow-white__img,
+  .arrow-white__scrimg {
+    display: none;
+  }
+
+  .shop__title {
+    font-size: 30px;
+    line-height: 29px;
+    margin: 0 14px 25px 0;
+  }
+
+  .shop-wrap {
+    height: 320px;
+    margin-bottom: 77px;
+    justify-content: space-evenly;
+  }
+
+  .sneaker-item {
+    width: 281px;
+  }
+
+  .sneaker-item__block {
+    width: 230px;
+    height: 233px;
+    margin-bottom: 23px;
+  }
+
+  .sneaker-item__name {
+    font-size: 20px;
+    line-height: 19px;
+    padding: 0 0 0 15px;
+    margin-bottom: 18px;
+  }
+
+  .sneaker-item__price {
+    font-size: 25px;
+    line-height: 24px;
+    padding: 0 0 0 15px;
+    margin: 0;
+  }
+
+  .colour-circle {
+    width: 94px;
+    padding: 0 8px 0 0;
+  }
+
+  .colour-circle__item {
+    width: 20px;
+    height: 20px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/axion_ann/pullRequest?prid=5730635)